pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,42 +14,66 @@
 
 package com.google.sps.servlets;
 
-import java.util.ArrayList;  
-import java.util.List;  
-import java.util.Date;  
-import com.google.gson.Gson;
+import java.util.ArrayList; 
+import java.util.Date; 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    
+  ArrayList<String> comments = new ArrayList<>();  
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    Comment prueba = new Comment("Title", "Msg", new Date(System.currentTimeMillis()));
+    response.setContentType("application/json;");
+    response.getWriter().println(comments);
```

Comment:
You need to convert the whole `comments` array to JSON. Otherwise I think JS won't be able to parse it.

Reply:
Actually... maybe it's fine like that

Reply:
The “/data” after posting comments looks like this:
[
 { subject : “subject”,
   msg: “msg”,
   date: date
  }
  { subject : “subject2”,
   msg: “msg2”,
   date: date2
  }, etc.
]
Is that ok?

Reply:
Yeah that's what you want. I tried it the way I said (incorrectly) and JS parses it as an array of strings rather than array of objects.

Reply:
Oh okay perfect so I'll leave it as it is :)